django-verdant/rca/static/rca/css/streamfield-blocks/two-column-block.less

```diff
@@ -0,0 +1,41 @@
+.two-column-block {
+    @media @media-desktop-small {
+        display: flex;
+    }
+
+    &__item {
+        flex-basis: 50%;
```

Comment:
mobile styling will have these stretch all the way across won't it?

Reply:
this won't have any effect until `display: flex` is set at 768px